asapdiscovery-cli/asapdiscovery/cli/cli_args.py

```diff
@@ -54,8 +54,17 @@ def dask_type(func):
     )(func)
 
 
+def dask_failure_mode(func):
+    return click.option(
+        "--dask-failure-mode",
+        type=click.Choice(DaskFailureMode.get_values(), case_sensitive=False),
+        default=DaskFailureMode.SKIP,
+        help="The failure mode for dask. Can be 'raise' or 'skip'.",
```

Comment:
maybe we should add show default to make it clear if the user needs to use the option or not?